### PR TITLE
Downgrade jackson2 version from 2.7.0. to 2.4.5

### PR DESCRIPTION
**Pull request by andreaturli**

### CODE REVIEW

File: rest/rest-api/src/main/java/org/apache/brooklyn/rest/domain/TaskSummary.java [hunk 1]
```diff
@@ -82,8 +82,8 @@ public TaskSummary(
             @JsonProperty("endTimeUtc") Long endTimeUtc,
             @JsonProperty("currentStatus") String currentStatus,
             @JsonProperty("result") Object result,
-            @JsonProperty("error") boolean error,
-            @JsonProperty("cancelled") boolean cancelled,
+            @JsonProperty("IsError") boolean error,
```
Comment (neykov) [inline rest/rest-api/src/main/java/org/apache/brooklyn/rest/domain/TaskSummary.java]: should be `isError`?

File: rest/rest-api/src/main/java/org/apache/brooklyn/rest/domain/EntityDetail.java [hunk 2]
```diff
@@ -50,17 +55,25 @@ public EntityDetail(
...
+        super(id, name, type, catalogItemId, Collections.<String, URI>emptyMap());
+        this.applicationId = applicationId;
+        this.iconUrl = iconUrl;
         this.parentId = parentId;
         this.children = (children == null) ? ImmutableList.<EntitySummary>of() : ImmutableList.copyOf(children);
         this.groupIds = (groupIds == null) ? ImmutableList.<String>of() : ImmutableList.copyOf(groupIds);
-        this.members = (members == null) ? ImmutableList.<Map<String,String>>of() : ImmutableList.copyOf(members);
+        this.members = (members == null) ? ImmutableList.<Map<String, String>>of() : ImmutableList.copyOf(members);
+        this.serviceState = serviceState;
+        this.serviceUp = serviceUp;
```
Comment (neykov) [inline rest/rest-api/src/main/java/org/apache/brooklyn/rest/domain/EntityDetail.java]: oops how did this happen :) good find!

File: pom.xml [hunk 3]
```diff
@@ -97,7 +97,7 @@
         <slf4j.version>1.6.6</slf4j.version>  <!-- used for java.util.logging jul-to-slf4j interception -->
         <guava.version>17.0</guava.version>
         <xstream.version>1.4.7</xstream.version>
-        <fasterxml.jackson.version>2.7.0</fasterxml.jackson.version>  <!-- more recent jackson, but not compatible with old annotations! -->
+        <fasterxml.jackson.version>2.4.5</fasterxml.jackson.version>  <!-- more recent jackson, but not compatible with old annotations! -->
```
Comment (sjcorbett) [inline pom.xml]: What about the resteasy dependency? With 3.0.14.FINAL we get this in brooklyn-tosca:
```
java.lang.NoSuchMethodError: com.fasterxml.jackson.databind.util.BeanUtil.okNameForSetter(Lcom/fasterxml/jackson/databind/introspect/AnnotatedMethod;Z)Ljava/lang/String;
        at com.fasterxml.jackson.module.jaxb.JaxbAnnotationIntrospector.findNameForDeserialization(JaxbAnnotationIntrospector.java:1004)
        at com.fasterxml.jackson.databind.introspect.AnnotationIntrospectorPair.findNameForDeserialization(AnnotationIntrospectorPair.java:577)
```
3.0.8.FINAL was the previous version.

File: rest/rest-api/src/main/java/org/apache/brooklyn/rest/domain/TaskSummary.java (hunk 1 above)
Comment (sjcorbett) [inline rest/rest-api/src/main/java/org/apache/brooklyn/rest/domain/TaskSummary.java]: This should be `isError`.

**Comment by sjcorbett** [comment]
I'm going to fix and merge ada47cf and 4158a31 in a separate pull request.

**Comment by andreaturli** [comment]
close in favor of #22 and #21

**Pull request closed without merging**